rust-src/concordium_base/src/curve_arithmetic/ed25519_instance.rs

```diff
@@ -318,7 +275,7 @@ pub(crate) mod tests {
             1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 254, 255, 255, 255, 255, 255, 255, 255,
             0, 0, 0, 0, 0, 0, 0, 0,
         ])
-        .expect("Expected a valid scalar")
+        .unwrap()
```

Comment:
This is strictly worse :) I wonder why it was changed?

Reply:
Yeah, they use constant-time option type `CtOption`, which does not have `expect()`, conversion it to `Option` requires some type annotations :)
I'll do this properly :)

Reply:
Fixed now.